src/sentry/api/bases/organization_events.py

```diff
@@ -525,6 +525,9 @@ def get_event_stats_data(
                             zerofill_results=zerofill_results,
                             dataset=dataset,
                         )
+                        results[key]["isMetricsExtractedData"] = self._query_if_extracted_data(
```

Comment:
I've moved the logic into a function with a try/except block to protect against side effects.